Source/Core/DolphinQt2/Host.cpp

```diff
@@ -53,6 +54,10 @@ bool Host::GetRenderFullscreen()
 void Host::SetRenderFullscreen(bool fullscreen)
 {
   m_render_fullscreen = fullscreen;
+
+  if (!g_renderer || g_renderer->IsFullscreen() == fullscreen ||
```

Comment:
This logic doesn't seem right to me. If there is no renderer, or the fullscreen state is already matching, call SetFullscreen? Should it be `if (g_ActiveConfig.ExclusiveFullscreenEnabled() && g_renderer && g_renderer->IsFullscreen() != fullscreen)`?

Reply:
Yep, fucked up a bit there.